packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -243,6 +243,16 @@ export const prodParasPolkadot: EndpointOption[] = [
       Pinknode: 'wss://public-rpc.pinknode.io/nodle'
     }
   },
+  {
+    info: 'omnibtc',
+    isUnreachable: true,
+    homepage: 'https://www.omnibtc.finance',
+    text: 'OmniBTC',
+    paraId: 2053,
+    providers: {
+      OmniBTC: 'wss://omnibtc.io/ws'
```

Comment:
We have 3 chains here with the exact same endpoint - this is not possible. It is either one of -

- a parachain
- a live network
- a test network

A single RPC cannot be linked to 3 different listings.

I would suggest just removing the entries in test networks and live networks and just keep it under Polkadot. (Which is where the chain lives)

Testing - these are solo test networks, running on their own
Production - these are solo production networks, running on their own